routes/orders.js

```diff
@@ -4,6 +4,8 @@ const ordersController = require("../controllers/ordersController.js")
 
 router.get("/", ordersController.getOrders);
 router.get("/:id", ordersController.getOrderById);
+router.get("/:id", ordersController.getOrderByDeliveryID);
+router.get("/:id", ordersController.getOrderByClientID);
```

Comment:
There are three GET requests with the same endpoint, all using a different Controller function. How does the backend know if the :id parameter is an order, user or delivery user id and route accordingly? I think the user related ones should become query parameters for the orders endpoint.

Reply:
ok!!